### ISSUE TITLE
kamon-redis: kamon.instrumentation.akka.filters.actors.trace.excludes is not picked

### ISSUE THREAD

**Issue by jtjeferreira** [kamon-io/Kamon#1072]
Hi!

I was trying the kamon-redis module with redisscala and everything is working fine, except a small detail regarding the traces:

![image](https://user-images.githubusercontent.com/943051/141322956-9d2bc255-127e-4af8-bff4-3e0cce1b6488.png)

Those `Tell` operations they are still present in the trace, although these traces are excluded in this code https://github.com/kamon-io/Kamon/blob/c6a32e743e74f055a383ba62c56fa64b29b03a2e/instrumentation/kamon-redis/src/main/resources/reference.conf#L21-L29 

In the REPL, I can see those configs are not picked:

```scala
scala> kamon.Kamon.config().getConfig("kamon.instrumentation.akka.filters.actors.trace")
val res7: com.typesafe.config.Config = Config(SimpleConfigObject({"excludes":[],"includes":["*/user/**","*/system/sharding**"]}))

scala> kamon.Kamon.config().getConfig("kanela.modules.redis")
val res8: com.typesafe.config.Config = Config(SimpleConfigObject({"description":"Provides tracing for Jedis, Lettuce and Rediscala libraries","instrumentations":["kamon.instrumentation.jedis.JedisInstrumentation","kamon.instrumentation.lettuce.LettuceInstrumentation","kamon.instrumentation.rediscala.RediscalaInstrumentation"],"name":"Redis Instrumentation","within":["redis.clients.jedis.Protocol","io.lettuce.core..*","redis..*"]}))
``` 

I suspect this is due to some loading order of the reference.conf files, because adding the above configuration to my `application.conf` file fixes the problem...

**Comment by ivantopo** [comment]
This seems to happen only when adding the dependencies one-by-one, instead of using the bundle. Fixing it in a bit.

**Comment by ivantopo** [comment]
This will get fixed with #1122 :smile:
